### ISSUE TITLE
BUG: Insufficient funds in batch payments returns successful message.

### ISSUE THREAD

**Issue by RamirezAlex** [kin-labs/kinetic#249]
```
{
  "id": "cl4eatz9q13039wp4zkd2tdqt",
  "createdAt": "2022-06-14T15:08:16.718Z",
  "updatedAt": "2022-06-14T15:08:17.344Z",
  "amount": null,
  "commitment": "Confirmed",
  "destination": null,
  "feePayer": null,
  "mint": null,
  "referenceId": "1655219295730",
  "referenceType": "DemoPayment",
  "signature": null,
  "solanaCommitted": null,
  "solanaConfirmed": null,
  "solanaFinalized": null,
  "solanaStart": null,
  "solanaTransaction": null,
  "source": null,
  "status": "Processing",
  "webhookEventStart": "2022-06-14T15:08:17.297Z",
  "webhookEventEnd": "2022-06-14T15:08:17.342Z",
  "webhookVerifyStart": null,
  "webhookVerifyEnd": null,
  "appEnvId": "cl4by8kj80158jrp4avtc95e0",
  "errors": []
}
```

**Comment by beeman** [comment]
I think this is also not properly handled on the normal `makeTransfer` endpoint, so seems like a good thing to add in our sdk-e2e tests.

**Comment by RamirezAlex** [comment]
Closed by #266
